### PR TITLE
feat(oakItems): add ability to rename loadouts

### LINKED ISSUE #2377: [SUGGESTION] Renaming loadout slots
Renaming loadout slots

**Current Behavior**
Loadouts are named 'Loadout 1', 'Loadout 2' and 'Loadout 3'....

**Improved Behavior**
Being able to rename our loadouts like 'Farming', 'Mining', 'Shiny hunting'...

**Reason for change**
Improve clarity in our items management. Stop wasting time and get confused.

**Detailed Description**
A simple gear or a pencil icon inside or near the loadout box would suffice.

Please consider.

### PR DESCRIPTION
**Pull request by shallotmama**
Closes #2377.

Adds a `Name` field to the loadout edit menu and displays the name on hover for the quick-switch buttons.

### CODE REVIEW

File: src/components/profileModal.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h5 class="modal-title">Profile</h5>
                 </button>
             </div>
             <div class="modal-body text-left p-2">
-                <div class="form-row p-0" id="breeding-filter">
+                <div class="form-row p-0" id="profileEdit">
```
Comment (shallotmama) [inline src/components/profileModal.html:11]: Noticed this when I was looking for something to copy-paste 😛

**Comment by Jaaslet** [comment]
I can't load this PR
image

**Comment by Aegyo** [comment]
@Jaaslet, I've updated this PR with some fixes

**Comment by shallotmama** [comment]
I've been moving, so I wasn't able to respond/make changes. Thanks @Aegyo!

**Review by Jaaslet — APPROVED** [review]
Looks good. I have updated the update function to the next version.
Thanks for the PR!

**Pull request merged**